### PR TITLE
I103 delete dialog

### LINKED ISSUE #103: Delete Dialog
## Basic Information
- Create a popup delete dialog that will show up when a delete button is clicked, if "yes" is clicked it can continue to run the delete function. 
- Make it into its own component
- Could also be used for "Accept" and "Reject" on the sign up requests page

Screen Shot 2022-12-17 at 10 06 13 pm

### PR DESCRIPTION
**Pull request by LemonTheSour**
## Change Summary
Added a Delete Dialog component using the existing modal component. Emits accept/reject events which can be used to trigger event handlers. 

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
closes #103

# Related issue

- Resolve #103

### CODE REVIEW

File: components/DeleteDialog.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          @click="emit('accept')"
+        />
+        <Button
+          class="font-semibold"
+          button-text="Reject"
+          button-color="bg-light-red"
+          text-color="text-dark-red"
+          size="medium"
+          @click="emit('reject')"
+        />
```
Comment (ChantelleKerr) [inline components/DeleteDialog.vue:24]: Change to Yes & No :D

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Nice job! It works well and looks great. One small change :D

File: components/DeleteDialog.vue (hunk 1 above)
Comment (ChantelleKerr) [inline components/DeleteDialog.vue:24]: I meant the button-text. So replace `button-text="Accept"` => `button-text="Yes"` & `button-text="Reject"` => `button-text="No"` :D

File: .env.example [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-# on how nuxt uses these variables.
-
-# FIREBASE =======================================
-NUXT_PUBLIC_FIREBASE_API_KEY=""
-NUXT_PUBLIC_FIREBASE_AUTH_DOMAIN=""
-NUXT_PUBLIC_FIREBASE_PROJECT_ID=""
-NUXT_PUBLIC_FIREBASE_STORAGE_BUCKET=""
-NUXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=""
-NUXT_PUBLIC_FIREBASE_APP_ID=""
-NUXT_PUBLIC_FIREBASE_MODE=""
```
Comment (ChantelleKerr) [inline .env.example:13]: This file needs to stay in for future developer so they know what keys are need in the proper .env file

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Sorry there was a slight misunderstanding and a removed file that shouldnt be removed. Thanks :D

File: components/DeleteDialog.vue (hunk 1 above)
Comment (LemonTheSour) [reply components/DeleteDialog.vue]: Thats so weird, on my version it says I made thoses changes, and when I try to push it says up to date, but on the github it clearly says accept/reject. Made it work, dunno what happened there

**Review by ChantelleKerr — APPROVED** [review]
Great job! Works well. Thanks for doing the changes

**Pull request merged**